internal/mode/static/telemetry/collector_test.go

```diff
@@ -150,7 +165,6 @@ var _ = Describe("Collector", Ordered, func() {
 				secret1 := &v1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "secret1"}}
 				secret2 := &v1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "secret2"}}
 				nilsecret := &v1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "nilsecret"}}
-
```

Comment:
nit: can you add this space back, just to keep these variables separated and a little more clear

Reply:
This hasn't been committed

Reply:
i apologise, confused it with the line between gateway and service. pushing a fix